package.json

```diff
@@ -176,9 +175,6 @@
     "inquirer": "3.3.0",
     "inquirer-confirm": "2.0.3",
     "jest": "24.9.0",
-    "js-codemod": "cpojer/js-codemod",
-    "jscodemods": "https://github.com/cypress-io/jscodemods.git#01b546e",
-    "jscodeshift": "0.7.0",
```

Comment:
Removing these 3 historical deps got rid of 249 transitive dependencies and 74 MB:

Before:

```
~ echo **/node_modules/* | wc -w
7371
~ du -c -B M **/node_modules/*
...
2792M   total
```

After:

```
~ echo **/node_modules/* | wc -w
7122
~ du -c -B M **/node_modules/*
...
2718M   total
```

Reply:
Nice!

I wonder if there's a tool we can use to detect unused dependencies and prune more?

Reply:
Yeah, it's called depcheck: https://github.com/depcheck/depcheck I want to add this to the repo at some point for sure.